File: codes/HanoiTowerSolu/HanoiTowerSolu.py
def HanoiTowerMove(num, fromm, by, to):
    if num == 1:
        print("원반1을 "+str(fromm)+"에서 "+str(to)+"로 이동")
    else:
        HanoiTowerMove(num-1, fromm, to, by)
        print("원반"+str(num)+"을(를) "+str(fromm)+"에서 "+str(to)+"로 이동")
        HanoiTowerMove(num-1, by, fromm, to)

HanoiTowerMove(3, 'A', 'B', 'C')

File: codes/HanoiTowerSolu/HanoiTowerSolu.java
class HanoiTowerMove{
    public void HanoiTowerMove(int num, char from, char by, char to){
        if(num == 1) {
            System.out.println("원반1을 " + from + "에서 " + to + "로 이동");
        } else {
            HanoiTowerMove(num-1, from, to, by);
            System.out.println("원반"+num+"을(를) "+from+"에서 "+to+"로 이동");
            HanoiTowerMove(num-1, by, from, to);
        }
    }
}

public class HanoiTowerSolu {
    public static void main(String[] agrs) {
        HanoiTowerMove htm = new HanoiTowerMove();
        htm.HanoiTowerMove(3, 'A', 'B', 'C');
    }
}
